packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -303,15 +303,18 @@ contract ValidatorManager is ISemver, IValidatorManager {
         ASSET_MANAGER.bondValidatorKro(validator);
     }
 
+    /**
+     * @inheritdoc IValidatorManager
+     */
+    function unbondValidatorKro(address validator) external onlyColosseum {
+        ASSET_MANAGER.unbondValidatorKro(validator);
+    }
+
     /**
      * @inheritdoc IValidatorManager
      */
     function slash(uint256 outputIndex, address winner, address loser) external onlyColosseum {
-        (uint128 tax, uint128 challengeReward) = ASSET_MANAGER.modifyBalanceWithChallenge(
-            loser,
-            0,
-            true
-        );
+        (uint128 tax, uint128 challengeReward) = ASSET_MANAGER.decreaseBalanceWithChallenge(loser);
 
         emit Slashed(outputIndex, loser, tax + challengeReward);
```

Comment:
`BOND_AMOUNT` should be fetched from `ASSET_MANAGER`, and the logic to calculate challenge reward and tax can be changed in the future. I thought it could be robust regardless of the logic in `ASSET_MANAGER` in this way. wdyt?

Reply:
That's valid, then let's leave this as it is.